src/client/components/ShoppingCart/ShoppingCart.styles.css

```diff
@@ -119,3 +131,94 @@
   justify-content: normal;
   font-weight: bolder;
 }
+.cart-detail-container{
+  display: flex;
+  box-shadow: 2px 2px 10px 5px rgba(0, 0, 0, 0.1);
+  
+  background-color:beige;
+  
+  height:fit-content;
+  
+  flex-direction: column;
+  
+  text-align: center;
+  color: black;
+  font-family: Arial, Helvetica, sans-serif;
+  
+}
+.cart-detail-container>div{
+margin: 5px;
+width:200px;
+color: black;
+
+}
+.proceed-payment{
+  background-color: #687804;
+  margin: 15px;
+
+}
+.right-div{
+  display: flex;
+  flex-direction: column;
+  font-weight: bolder;
+  
+}
+.right-div>div{
+  margin :10px;
+  
+}
+
+.continue-shop{
+  background-color: #687804;
+  margin:20px;
+  
+}
+.parent-div{
+  display: flex;
+  flex-direction: column;
+  border: 2px solid red;
+  
+
+}
+.parent-div > div{
+  margin:15px;
+
+}
+.user-detail{
+
+background-color:beige;
+padding: 20px;
+
+color: black;
+font-weight: bolder;
+box-shadow: 2px 2px 10px 5px rgba(0, 0, 0, 0.1);
```

Comment:
We don't need this box-shadow style. Its not even there in the Figma reference!
Suggest you remove that, as it is not looking in sync.